src/kernel/include/fpu.h

```diff
@@ -105,12 +104,12 @@ fpu_init(void)
 		return -1;
 	}
 #endif
-
 	fpu_set(FPU_DISABLE);
+	printk("\n\n\n FPU_DISABLED");
 	*PERCPU_GET(fpu_disabled)  = 1;
 	*PERCPU_GET(fpu_last_used) = NULL;
 
-	/* printk("fpu_init on core %d\n", get_cpuid()); */
+	printk("fpu_init on core %d\n", get_cpuid());
```

Comment:
I'll echo @WenyuanShao 's comment: we don't want any printing in the final code aside from some printouts at boot time to say that we're using FPU/lazy FPU.

Reply:
Got it, I will delete all the related printouts.